lib/routes/visionias/utils.ts

```diff
@@ -0,0 +1,107 @@
+import { getCurrentPath } from '@/utils/helpers';
+const __dirname = getCurrentPath(import.meta.url);
+
+import { DataItem } from '@/types';
+import { parseDate } from '@/utils/parse-date';
+import { art } from '@/utils/render';
+import path from 'node:path';
+import ofetch from '@/utils/ofetch';
+import { load } from 'cheerio';
+
+export const baseUrl = 'https://visionias.in';
+
+export async function extractNews(item, selector) {
```

Comment:
Please [cache](https://docs.rsshub.app/joinus/advanced/use-cache) the returned object